### PR TITLE
update website for release-2.48.0 and blog post

### PR DESCRIPTION
**Pull request by riteshghorse**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: website/www/site/content/en/blog/beam-2.48.0.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+We are happy to present the new 2.48.0 release of Beam.
+This release includes both improvements and new functionality.
+See the [download page](/get-started/downloads/{$DOWNLOAD_ANCHOR}) for this release.
+
+<{$REMOVE_FOR_VALID_SUMMARY_BREAK}!--more-->
```
Comment (damccorm) [inline website/www/site/content/en/blog/beam-2.48.0.md:26]:
```suggestion
<!--more-->
```

**Review by damccorm — APPROVED** [review]
LGTM (other than one nit), of course shouldn't be merged until:
1) release is complete
2) dates are updated to match actual release date

**Comment by riteshghorse** [comment]
Updated the data, merging now
Thanks!

**Comment by Abacn** [comment]
website tests and deploy failing since this PR, error:

```
20:16:24 ERROR 2023/06/01 00:16:24 Author 'riteshghorse' is not defined. Please add them to 'website/www/site/data/authors.yml'
20:16:35 Total in 12880 ms
20:16:35 Error: Error building site: logged 1 error(s)
20:16:35 error Command failed with exit code 255.
20:16:35 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

See https://ci-beam.apache.org/view/PostCommit/job/beam_PreCommit_Website_Cron/6857/console

**Pull request merged**